Packs/GitHub/Integrations/GitHub/GitHub.py

```diff
@@ -1333,28 +1434,56 @@ def fetch_incidents_command():
     'GitHub-create-pull-request': create_pull_request_command,
     'Github-get-github-actions-usage': get_github_actions_usage,
     'GitHub-get-file-content': get_file_content_from_repo,
+    'GitHub-search-code': search_code_command,
 }
 
-'''EXECUTION'''
 
-if TOKEN == '' and PRIVATE_KEY != '':
-    try:
-        import jwt
-    except Exception:
-        return_error("You need to update the docket image so that the jwt package could be used")
+def main():
+    global USER
+    global TOKEN
+    global PRIVATE_KEY
+    global INTEGRATION_ID
+    global INSTALLATION_ID
+    global REPOSITORY
+    global USE_SSL
+    global FETCH_TIME
+    global USER_SUFFIX
+    global ISSUE_SUFFIX
+    global RELEASE_SUFFIX
+    global PULLS_SUFFIX
+    global HEADERS
+
+    USER = demisto.params().get('user')
+    TOKEN = demisto.params().get('token', '')
+    creds: dict = demisto.params().get('credentials', {})
+    PRIVATE_KEY = creds.get('sshkey', '') if creds else ''
+    INTEGRATION_ID = demisto.params().get('integration_id')
+    INSTALLATION_ID = demisto.params().get('installation_id')
+    REPOSITORY = demisto.params().get('repository')
+    USE_SSL = not demisto.params().get('insecure', False)
+    FETCH_TIME = demisto.params().get('fetch_time', '3')
```

Comment:
since you already modified it...
```suggestion
    params = demisto.params()
    USER = params.get('user')
    TOKEN = params.get('token', '')
    creds: dict = params.get('credentials', {})
    PRIVATE_KEY = creds.get('sshkey', '') if creds else ''
    INTEGRATION_ID = params.get('integration_id')
    INSTALLATION_ID = params.get('installation_id')
    REPOSITORY = params.get('repository')
    USE_SSL = not params.get('insecure', False)
    FETCH_TIME = params.get('fetch_time', '3')
```